python/tank/util/shotgun.py

```diff
@@ -264,9 +264,7 @@ def get_associated_sg_base_url():
     
     :returns: The base url for the associated Shotgun site
     """
-    cfg = __get_sg_config()
-    config_data = __get_sg_config_data(cfg)
-    return config_data["host"]
+    return get_associated_sg_config_data().get("host", "")
```

Comment:
Wondering about the empty string by default. I think the old code requires the config value to be present and otherwise raises an exception. Any reason not to keep this behaviour?